closed/autoconf/custom-hook.m4

```diff
@@ -349,6 +349,14 @@ AC_DEFUN([OPENJ9_CONFIGURE_JITSERVER],
 
   AC_MSG_CHECKING([for jitserver])
   OPENJ9_ENABLE_JITSERVER=false
+
+  case "$OPENJ9_PLATFORM_CODE" in
+     xa64|xl64|xz64)
+        if test "x$enable_jitserver" = x; then
+           enable_jitserver=yes
+        fi
+       ;;
+  esac
```

Comment:
Please move this into the branch that handles the default case (merging with (old) line 363.
As is, configure will incorrectly report that JITServer was explicitly enabled.
Please follow the surrounding formatting (indent two spaces per level, space before `;` that terminates `if` expression).

Reply:
For clarity, I'd expect the default branch to become something like this:
```
    case "$OPENJ9_PLATFORM_CODE" in
      xa64|xl64|xz64)
        AC_MSG_RESULT([yes (default)])
        OPENJ9_ENABLE_JITSERVER=true
        ;;
      *)
        AC_MSG_RESULT([no (default)])
        ;;
      esac
```

Reply:
I've made your requested changes in d77faa9a33799630e46246f66d6a6b8e1c1f212d. Please take a look.